src/UI/Components/ServiceInstanceForm/Helpers/FieldCreator.ts

```diff
@@ -229,6 +229,25 @@ export class FieldCreator {
           };
         }
 
+        // WORKAROUND TO ADD SUPPORT FOR TEXTAREA
+        if (
+          (attribute.type === "string" || attribute.type === "string?") &&
+          (attribute.validation_type === "pydantic.constr" ||
+            attribute.validation_type === "pydantic.constr?") &&
+          attribute.validation_parameters.max_length === 10000
```

Comment:
max_length will be always 10000?

Reply:
it's the workaround we agreed on, it's until there is a better implementation on the BE for these :D

Reply:
Oh, that was behind workaround, now I understand :D

Reply:
is it 10000 or is it longer than 255?

Reply:
I made the adjustment !